cli/src/main.rs

```diff
@@ -347,6 +446,7 @@ use codespan_reporting::{
         termcolor::{ColorChoice, StandardStream},
     },
 };
+use naga::back::{glsl::Version, hlsl::ShaderModel, IndexBoundsCheckPolicy};
```

Comment:
If this is needed for the code above, let's have it at the top.
Actually, do we need to include `glsl::Version` and `hlsl::ShaderModel`? We generally avoid including backend-specific things.